openqabot/loader/config.py

```diff
@@ -64,16 +64,16 @@ def read_products(path: Path) -> List[Data]:
         data = loader.load(p)
 
         if not data:
-            logger.warning("something wrong with %s" % str(p))
+            logger.debug("Skiping invalid config %s" % str(p))
```

Comment:
I suggest to not simply reuse the same text as in before but use specific, differing log messages

Reply:
```suggestion
            logger.debug("Skipping invalid config %s" % str(p))
```